packages/codemirror-extension/src/index.ts

```diff
@@ -164,6 +169,22 @@ function activateEditorCommands(
     if (keyMap === 'vim') {
       // @ts-expect-error
       await import('codemirror/keymap/vim.js');
+      commands.addKeyBinding({
```

Comment:
Would it be easy to add a `signal.emit` here with the keymap in it? I could see that being helpful for extensions building around the keymaps. If not easy then it's no big deal.